resources/web/wwi/x3d.js

```diff
@@ -134,6 +134,7 @@ THREE.X3DLoader = class X3DLoader {
   parseTransform(transform) {
     var object = new THREE.Object3D();
     object.userData.x3dType = 'Transform';
+    object.userData.id = getNodeAttribute(transform, 'id', '');
```

Comment:
The `id` is already stored in the `object.name` field and there is a function to retrieve the object by this value.
So this is a duplication of the same data, why do you need it?

Reply:
Thank you, I was not aware of this. I will look if I could reuse `object.name`.

Reply:
I could not use the `userData` field nor the `Object3D.traverse` because these are only available for THREE.Object3D nodes but not for others nodes, like the THREE.Texture.

Reply:
Using `object.name` and `getObjectByCustomId()` is indeed much better. I take the advantage of this to rename this function and change its parameters order (to simplify the calls):

```
getObjectById(id, object=undefined, context=undefined)
```